apriltag:  # node name
  ros__parameters:
    image_transport: raw
    family: 36h11
    # size: 0.13             # 默认 tag 边长（不推荐使用，推荐用 tag.sizes 单独设置）
    max_hamming: 0

    detector:
      threads: 1
      decimate: 2.0
      blur: 0.0
      refine: True
      sharpening: 0.25
      debug: False 

    pose_estimation_method: "pnp"

    tag:
      ids:    [0 , 1 , 2 , 3 , 4]
      frames: [tag_0 , tag_1 , tag_2 , tag_3 , tag_4]
      sizes:  [0.13 , 0.13 , 0.13 , 0.13 , 0.13]

# tag:
#   ids: [9, 14]            # tag ID
#   frames: [base, object]  # optional frame name
#   sizes: [0.162, 0.162]   # optional tag-specific edge size
